app/domain/errors.rb

```diff
@@ -401,6 +401,18 @@ module AuthnJwt
               "   with field in the decoded token that contains the identity",
         code: "CONJ00075E"
       )
+
+      InvalidUriConfiguration = ::Util::TrackableErrorClass.new(
+        msg: "Uri authenticator configuration is invalid. It should configured as authenticator variables: " \
+              "one of the following: '{0-resource-name}','{1-resource-name}'",
+        code: "CONJ00070E"
+      )
```

Comment:
in case @shulifink  agree with me on the troubleshooting section, this should be there too.